components/action-mgt/org.wso2.carbon.identity.action.execution/src/test/java/org/wso2/carbon/identity/action/execution/util/APIClientTest.java

```diff
@@ -129,7 +132,7 @@ public void testCallAPIUnacceptableContentTypeForSuccessResponse()
     @DataProvider(name = "unacceptableSuccessResponsePayloads")
     public String[] unacceptableSuccessResponsePayloads() {
 
-        return new String[]{"{}", "", "success", "{\"actionStatus\":\"SUCCESS\"}", "{\"actionStatus\":\"ERROR\"}, " +
+        return new String[]{"{}", "", "success", "{\"actionStatus\":\"ERROR\"}, " +
```

Comment:
When it comes to Success responses IMO we should validate if at least either data or operations are available. Given this is related to extensions, just accepting success doesn't make sense. That is rather an event than an extension

Reply:
There can be such scenrios. Eg: In pre-issue access token action execution, there can be scenario where performable opertion need to be perform only if condition statisfied.